platform/linux-generic/m4/odp_pcap.m4

```diff
@@ -9,7 +9,7 @@ AC_CHECK_HEADER(pcap/pcap.h,
 [])
 
 if test "$have_pcap" = "yes"; then
-    ODP_CFLAGS="$AM_CFLAGS -DHAVE_PCAP"
+    AC_DEFINE([HAVE_PCAP], 1, [Define to 1 if you have pcap library])
```

Comment:
Compare this with other descriptions in config.h: 
- `Define to 1 if you have the ANSI C header files`
- `Define to 'int' if <sys/types.h> does not define.`
I think mine version is more in style with them.